matchtree.go

```diff
@@ -888,11 +897,93 @@ func (d *indexData) newMatchTree(q query.Q) (matchTree, error) {
 			all:       regexp.String() == "(?i)(?-s:.)*",
 			matchTree: subMT,
 		}, nil
+
+	case *query.RepoBranches:
+		reposBranchesWant := make(map[uint16]uint64)
```

Comment:
what about avoiding the map?
```suggestion
		reposBranchesWant := make([]uint64, len(repoMetaData))
```

Reply:
done